config/sunspot.yml

```diff
@@ -0,0 +1,25 @@
+production:
+  solr:
+    hostname: localhost
+    port: 8983
+    log_level: WARNING
+    path: /solr/production
+    # read_timeout: 2
+    # open_timeout: 0.5
+
+development:
+  solr:
+    hostname: localhost
+    port: 8982
+    log_level: INFO
+    path: /solr/development
+
+test: &TEST
+  solr:
+    hostname: localhost
+    port: 8981
+    log_level: WARNING
+    path: /solr/test
+
+cucumber:
```

Comment:
cucumber is not actually an environment: cucumber tests run in the test environment. was this line generated or added after?